packages/kbn-search-index-documents/components/result/rich_result_header.tsx

```diff
@@ -85,9 +88,10 @@ const MetadataPopover: React.FC<MetaDataProps> = ({
   return (
     <EuiPopover button={metaDataIcon} isOpen={popoverIsOpen} closePopover={closePopover}>
       <EuiPopoverTitle>
-        {i18n.translate('searchIndexDocuments.result.header.metadata.title', {
-          defaultMessage: 'Document metadata',
-        })}
+        <FormattedMessage
```

Comment:
Changed to use `FormattedMessage` and changed id since same id used in [two other places currently](https://github.com/search?q=repo%3Aelastic%2Fkibana%20searchIndexDocuments.result.header.metadata.deleteDocument&type=code).